### ISSUE TITLE
Implement & Test Various Loss Functions

### ISSUE THREAD

**Issue by andreicozma1** [aicip/SatMAE#16]
So far we have
- MSE
- L1

To investigate:
- Cross-Entropy
- KL
- Various combinations

Goal is to hopefully enhance reconstruction of structure/details a little bit.

**Comment by andreicozma1** [comment]
![image](https://user-images.githubusercontent.com/14914491/224899581-a9a41339-3a48-449a-bbb7-2b4ea2196834.png)
